.github/workflows/e2e.yml

```diff
@@ -87,16 +87,16 @@ jobs:
       - uses: pnpm/action-setup@c3b53f6a16e57305370b4ae5a540c2077a1d50dd # pin@v2.2.4
         with:
           version: 7
-      - name: Build validator for devnet
-        run: cargo build --bin sui-test-validator --profile dev
 
-      # checkout and build the current branch
-      - uses: actions/checkout@7dd9e2a3dc350cf687eb1b2a4fadfee8c8e49675 # pin@v3
```

Comment:
you need to check out back to the current branch after the downloading because we want to run the latest ts code against the binaries on devnet branch